graphql-database-dynmodb-history-lambda/src/main/java/com/fleetpin/graphql/database/dynamo/history/lambda/HistoryLambda.java

```diff
@@ -0,0 +1,61 @@
+package com.fleetpin.graphql.database.dynamo.history.lambda;
+
+import static java.util.stream.Collectors.groupingBy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.DynamodbEvent;
+import com.fleetpin.graphql.database.manager.dynamo.HistoryUtil;
+
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient;
+import software.amazon.awssdk.services.dynamodb.model.Record;
+import software.amazon.awssdk.services.dynamodb.model.WriteRequest;
+
+public abstract class HistoryLambda implements RequestHandler<DynamodbEvent, Void> {
+
+	public HistoryLambda() {
+
+	}
+
+	public abstract String getTableName();
+
+	public abstract DynamoDbAsyncClient getClient();
+
+	@Override
+	public Void handleRequest(DynamodbEvent input, Context context) {
+		var records = input.getRecords().stream().map(DynamoUtil::toV2);
+		process(records);
+		return null;
+	}

```

Comment:
Would be nice to have the output for potential debugging purposes.

Would require doing a little bit of gross CF coercing though.

Reply:
risk of outputting is the cloud watch cost will greatly exceed the lambda processing charge.